web/packages/teleterm/src/ui/services/resources/resourcesService.ts

```diff
@@ -142,6 +145,23 @@ export class ResourceSearchError extends Error {
     this.clusterUri = clusterUri;
     this.resourceKind = resourceKind;
   }
+
+  messageWithClusterName(
```

Comment:
This method will be called later in the ActionPicker too, that's why I defined it on the error class.